cypress/integration/Case1_Sign_up.js

```diff
@@ -6,89 +6,87 @@ import HomePage from '../PageObjects/HomePage'
 // import RewardsPage from '../PageObjects/RewardsPage'
 
 describe('Test case 1: Ability to Sign Up', () => {
-  it('User is not able to sign up the wallet with wrong values', () => {
-    cy.visit(Cypress.env('baseUrl') + '?paymentCode=1234567asQ3')
-    StartPage.continueOnWebButton.click()
-    StartPage.createWalletButton.click()
-    SignUpPage.pageHeader.should('contain', 'Sign Up')
-    SignUpPage.nameInput.should('be.visible')
-    SignUpPage.nextButton.should('be.visible')
-
-    SignUpPage.nameInput.type('Name')
-    SignUpPage.invalidValueErrorMessage1.should('exist')
-    SignUpPage.nameInput.clear()
-    SignUpPage.nameInput.type('Name1 Name')
-    SignUpPage.invalidValueErrorMessage2.should('exist')
-    SignUpPage.nameInput.clear()
-    SignUpPage.nameInput.type(Cypress.env('usernameForRegistration'))
-    SignUpPage.nextButton.should('have.attr', 'data-focusable')
-    SignUpPage.nextButton.click()
-    SignUpPage.phoneInput.type('+11111')
-    SignUpPage.invalidValueErrorMessage3.should('exist')
-    SignUpPage.phoneInput.clear()
-    SignUpPage.phoneInput.type('+9999999999999')
-    SignUpPage.invalidValueErrorMessage3.should('exist')
-  })
-
-  it('User is able to sign up the wallet with correct values', () => {
-    cy.visit(Cypress.env('baseUrl') + '?paymentCode=123', {
-      onBeforeLoad(win) {
-        delete win.navigator.__proto__.serviceWorker
-      },
+    it('User is not able to sign up the wallet with wrong values', () => {
+        cy.visit(Cypress.env('baseUrl') + '?paymentCode=1234567asQ3')
```

Comment:
paymentcode is not required, remove from all places except for where testing paymentlinks